package.nls.fr.json

```diff
@@ -64,5 +64,6 @@
 	"asciidoc.useWorkspaceRoot.deprecationMessage": "Ce paramètre a été remplacé par `#asciidoc.useWorkspaceRootAsBaseDirectory#` et n'a plus aucun effet.",
 	"asciidoc.use_kroki.desc": "**Obsolète:** Active l'extension Kroki afin de générer des diagrammes.",
 	"asciidoc.use_kroki.deprecationMessage": "Ce paramètre a été remplacé par `#asciidoc.extensions.enableKroki#` et n'a plus aucun effet.",
-	"asciidoc.antora.enableAntoraSupport.desc": "Active le support [Antora](https://antora.org/)."
-}
\ No newline at end of file
+	"asciidoc.antora.enableAntoraSupport.desc": "Active le support [Antora](https://antora.org/).",
+	"configuration.asciidoc.editor.drop.enabled": "Active/desactive le glisser-déposer dans l'editeur Asciidoc. Nécessite l'activation `#editor.dropIntoEditor.enabled#`."
```

Comment:
Using a `configuration` prefix is a good idea (to avoid conflicts with other translations) but we should do it globally otherwise it's inconsistent. Since that's beyond the scope of this pull request I would rather use  `asciidoc.editor.enableDrop.desc` and prepare another PR if we want to change the naming convention.